container-installs/docker-install.sh

```diff
@@ -35,22 +35,14 @@ else
 fi
 
 # Check if the keyrings directory exists & if not, create it
-if [ -f /etc/apt/keyrings/docker.asc ]
+if (test -f /etc/apt/keyrings/docker.asc )
```

Comment:
these changes do the same thing but good if you're going with some standard :+1: 

nit: either remove the space between `...asc` and ` )` or add one at the begining of `(test...` just for adhering to a style.
Just fyi:
`if [ ... ]...` needs spaces.
`if(...)` doesn't

Reply:
Ohh that makes sense, as to why I was having intermittent luck with
`[....]` thank you sir